daemon/start.go

```diff
@@ -36,9 +36,6 @@ func (daemon *Daemon) ContainerStart(name string, hostConfig *runconfig.HostConf
 				return err
 			}
 			container.Unlock()
-			if err := daemon.adaptContainerSettings(hostConfig, false); err != nil {
```

Comment:
I would be concerned that the sequence of calls has changed with Windows.  No clue what windows is doing, but if for example `InitDNSHostConfig` needed some change this would cause an issue.